### PR TITLE
update dependencies

### LINKED ISSUE #344: Update dependencies
We need to update the npm dependencies.

### PR DESCRIPTION
**Pull request by robbyoconnor**
Fixes #344

### CODE REVIEW

**Comment by jedireza** [comment]
FYI: Because there are not automated tests for Drywall (which is sad but true), I'll have to manually check a running system after we update the deps to make sure everything is working and not throwing exceptions.

File: package.json [hunk 1]
```diff
@@ -36,23 +36,23 @@
...
-    "grunt-contrib-uglify": "^0.9.x",
+    "grunt-contrib-uglify": "^0.11.x",
     "grunt-contrib-watch": "^0.6.x",
     "grunt-newer": "^1.x.x",
     "grunt-nodemon": "^0.4.x",
     "html5shiv": "^3.x.x",
     "jquery": "^2.x.x",
     "jquery.cookie": "^1.x.x",
-    "moment": "^2.x.x",
+    "moment": "^2.11.2",
```
Comment (jedireza) [inline package.json]: Let's remove all the `^` symbols. For modules that have a major version the versions should be `N.x.x`, and modules who's major version is `0` the versions should be `0.N.x`.

Comment (robbyoconnor) [inline package.json]: Fixed in bd3c136

**Comment by robbyoconnor** [comment]
At the very least -- update the the moment.js dependency...I doubt these should cause any issues...if they do -- I'll update the PR

**Comment by jedireza** [comment]
:smile: sorry I could've been more clear. Let's remove all the `^` characters in the whole `package.json` file. And be sure to use the version patterns as mentioned before:

> Let's remove all the `^` symbols. For modules that have a major version, the versions should be `N.x.x`, and modules who's major version is `0`, the versions should be `0.N.x`.

So the moment dependency should be `2.x.x` instead of `2.11.2`.

**Comment by robbyoconnor** [comment]
Addressed in 3f8bc7e

File: package.json [hunk 2]
```diff
@@ -7,52 +7,52 @@
...
-    "passport-tumblr": "^0.1.x",
-    "passport-twitter": "^1.x.x",
-    "serve-static": "^1.9.x",
-    "underscore": "^1.x.x"
+    "async": "1.x.x",
+    "backbone": "1.x.x",
+    "bcrypt": "0.8.x",
+    "body-parser": "1.x.x",
+    "compression": "1.x.x",
+    "connect-mongo": "1.1.x",
```
Comment (jedireza) [inline package.json]: 1.x.x

File: package.json [hunk 3]
```diff
@@ -7,52 +7,52 @@
...
+    "cookie-parser": "1.x.x",
+    "csurf": "1.x.x",
+    "emailjs": "1.0.x",
```
Comment (jedireza) [inline package.json]: 1.x.x

File: package.json [hunk 4]
```diff
@@ -7,52 +7,52 @@
...
+    "express": "4.12.x",
+    "express-session": "1.x.x",
+    "helmet": "1.1.x",
```
Comment (jedireza) [inline package.json]: 1.x.x

File: package.json [hunk 5]
```diff
@@ -7,52 +7,52 @@
...
+    "jade": "1.x.x",
+    "method-override": "2.x.x",
+    "mongoose": "4.x.x",
+    "morgan": "1.x.x",
+    "passport": "0.3.x",
+    "passport-facebook": "2.x.x",
+    "passport-github": "1.1.x",
```
Comment (jedireza) [inline package.json]: 1.x.x

File: package.json [hunk 6]
```diff
@@ -7,52 +7,52 @@
...
+    "passport-google-oauth": "1.0.x",
```
Comment (jedireza) [inline package.json]: 1.x.x

File: package.json [hunk 7]
```diff
@@ -7,52 +7,52 @@
...
-    "jquery.cookie": "^1.x.x",
-    "moment": "^2.x.x",
-    "respond.js": "^1.x.x"
+    "bootstrap": "3.x.x",
+    "david": "7.x.x",
+    "font-awesome": "4.x.x",
+    "grunt": "0.4.x",
+    "grunt-cli": "0.1.13",
+    "grunt-concurrent": "2.x.x",
+    "grunt-contrib-clean": "1.0.x",
```
Comment (jedireza) [inline package.json]: 1.x.x

File: package.json [hunk 8]
```diff
@@ -7,52 +7,52 @@
...
+    "grunt-contrib-copy": "0.8.x",
+    "grunt-contrib-jshint": "1.0.x",
```
Comment (jedireza) [inline package.json]: 1.x.x

**Comment by robbyoconnor** [comment]
Fixed in e2737deabdba6d4565e7482af98b19cc8d903a4b

**Comment by jedireza** [comment]
Sorry for the delay. I just ran through quickly and things seem to be working fine. Thanks again!

**Pull request merged**